proto/greptime/v1/region/server.proto

```diff
@@ -79,15 +79,17 @@ message CreateRequest {
   string engine = 2;
   // Columns in this region.
   repeated ColumnDef column_defs = 3;
-  // Id of columns in the primary key.
+  // Column Id of primary keys.
   repeated uint32 primary_key = 4;
   // Create region if not exists.
   bool create_if_not_exists = 5;
-  // Directory for region's data home. Usually is composed by catalog and table
-  // id
-  string region_dir = 6;
+  // catalog name and schema name is to accomplish storage path
+  // catalog name
+  string catalog = 6;
+  // schema name
+  string schema = 7;
```

Comment:
Does this also require to replace `region_dir` in `RegionCreateRequest` with `catalog/schema`?
https://github.com/GreptimeTeam/greptimedb/blob/5fbc9410230007cf5414ad4cd131c5de1832f2dc/src/store-api/src/region_request.rs#L138-L151

Reply:
This is not related to proto? That is defined separately